config/dependencies.yml

```diff
@@ -75,4 +75,7 @@ googleapis_common_protos:
 nodeModules:
   arguejs:
     version: '0.2.3'
-
+  extend:
+    version: '3.0.0'
+  lodash:
+    version: '4.6.0'
```

Comment:
I think it is in a quite similar position as 'googleapis_common_protos' section, so I added there.